<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create_moto</class>
 <widget class="QWidget" name="Create_moto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_createmoto">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Criar Moto</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>541</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QWidget" name="">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Modelo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_model"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Marca:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_brand"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Placa:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_plate"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Cor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_color"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Km Atual:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_atualkm"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Valor Pago:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_payedvalue"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Data Compra:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="date_buy">
       <property name="minimumDate">
        <date>
         <year>2022</year>
         <month>11</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
